src/main/mate.academy/dao/ValidatorImpl.java

```diff
@@ -0,0 +1,14 @@
+package dao;
+
+public class ValidatorImpl implements Validator {
+    private static final int MAX_LENGTH = 3;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public void validator(String[] fileDataLine) {
```

Comment:
maybe you should change the name of this parameter
```suggestion
    public void validator(String[] dataFromFile) {
```